blocks/image-collage/image-collage.css

```diff
@@ -41,6 +41,20 @@ main .image-collage[class*='boxy-col-']>div>div:first-of-type {
   padding-bottom: 0;
 }
 
+main .image-collage[class*='boxy-col-']:not(.autoblocked) > div > div:first-of-type {
```

Comment:
These fixed heights will crop the pictures in most cases, if that's what we want then fine but looking at the differences from the visual testing it doesn't seem that great

Reply:
I just left a comment for this line but I think all of the css rules that have a fixed height should get the `.crop-image` class

Reply:
@dnbute Add `.crop-image` to all fixed height, you can check